babel-standalone/build.boot

```diff
@@ -20,10 +20,12 @@
 
 (deftask package []
   (comp
-    (download :url (format "https://cdnjs.cloudflare.com/ajax/libs/babel-standalone/%s/babel.min.js" +lib-version+)
-              :checksum "F22EF4422B643D1FC3900CBAB4DB4D9D")
-    (sift :move {#"babel\.min\.js" "cljsjs/babel-standalone/production/babel.min.js"})
+    (download :url (format "https://cdnjs.cloudflare.com/ajax/libs/babel-standalone/%s/babel.js" +lib-version+))
+    (download :url (format "https://cdnjs.cloudflare.com/ajax/libs/babel-standalone/%s/babel.min.js" +lib-version+))
+    (sift :move {#"babel\.js" "cljsjs/babel-standalone/development/babel.inc.js"
+                 #"babel\.min\.js" "cljsjs/babel-standalone/production/babel.min.inc.js"})
```

Comment:
As you changed this path to include `.inc`, the Cljs namespace for js-transforms needs to also be updated: https://github.com/cljsjs/packages/blob/master/babel-standalone/src/cljsjs/babel_standalone.clj#L20